helm/azure-operator/ci/default-values.yaml

```diff
@@ -1,52 +1,49 @@
 ---
-Installation:
-  V1:
-    Name: ""
-    Registry:
-      Domain: ""
-    Guest:
-      Calico:
-        CIDR: ""
-        Subnet: ""
-      Docker:
-        CIDR: ""
-      Ingress:
-        Version: "0.9.0-beta.11"
-      Kubectl:
-        Version: "f51f93c30d27927d2b33122994c0929b3e6f2432"
-      IPAM:
-        NetworkCIDR: "10.1.0.0/8"
-        CIDRMask: 16
-      Kubernetes:
-        API:
-          Auth:
-            Provider:
-              OIDC:
-                ClientID: ""
-                IssuerURL: ""
-                UsernameClaim: ""
-                GroupsClaim: ""
-          ClusterIPRange: "172.31.0.0/16"
-          EndpointBase: "k8s.{{.Base}}"
-        ClusterDomain: "cluster.local"
-        IngressController:
-          BaseDomain: "k8s.{{.Base}}"
-      SSH:
-        SSOPublicKey: ""
-        UserList: ""
-    Secret:
-      AzureOperator:
-        SecretYaml: ""
-      Registry:
-        Dockerhub:
-          Token: ""
-    Provider:
-      Azure:
-        Cloud: ""
-        HostCluster:
-          CIDR: ""
-          VirtualNetwork: ""
-          VirtualNetworkGateway: ""
-        Location: ""
-        MSI:
-          Enabled: true
+azure:
```

Comment:
Shouldn't all of that go to `vaules.yaml`? I think we should get rid of the `ci` dir.

Reply:
Good idea, I'd rather do it in a separate PR.